applications/zkverse.md

```diff
@@ -0,0 +1,126 @@
+- Name of team member: Aaron
+
+### Contact
+
+- **Contact Name:** Bun  
+- **Contact Email:** csj1071285741@gmail.com  
+
+### Legal Structure
+
+- **Registered Address:** 5001 BEACH ROAD, #07-37, GOLDEN MILE COMPLEX
+SINGAPORE (199588)
+- **Registered Legal Entity:** SYN UNIVERSAL PRIVATE LTD
+
+### Team's experience
+- Bun
+    - He has many years of blockchain experience, is familiar with the underlying protocols, consensus algorithms, and common cryptographic algorithms of blockchain.
+    - He is mainly interested in zero-knowledge proof, post-quantum cryptography, etc
+    - He was used to be a member of [chainx](https://github.com/chainx-org/ChainX), mainly develop pallet
+    - Currently, he is a substrate ambassador jointly decided by oneblock+ community and parity
+
+
+### Team Code Repos
+- https://github.com/Zkvers/substrate-zk
+- https://github.com/Zkvers/snarkjs-bellman-adapter
+
+
+## Development Status :open_book:
+Currently, we have developed a substrate-based chain with zkp protocol and a practical zkp tool (which can adapt to two different zkp libraries to facilitate developers to develop zkp Dapps)
+
+## Development Roadmap :nut_and_bolt:
+We will implement two different proof systems(groth16 and plonk) separately to meet the needs of different developers
+
+### Overview
+
+- **Total Estimated Duration:** 2.5 months
+- **Full-Time Equivalent (FTE):**  1FTE
+- **Total Costs:** 30,000 USD
+
+### Milestone 1
+
+- **Estimated duration:** 1 month
+- **FTE:**  1
+- **Costs:** 13,000 USD
+
+Implement groth16 pallet in Substrate and develop a tool which can let the proof of snarkjs(off-chain) verified by the bellman library(on-chain). In this way, We can make it convenient for developers to develop zkp applications using the groth16(Bellman is an excellent zkp lib, but it is easy to make mistakes when using bellman to develop circuits, and snarkjs can cooperate with circom to write circuit with few mistakes. So we combined them to make the development of zkp applications more safe and efficient). So far, we haven't seen any team do this, but we think it is very meaningful.
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | License | Apache 2.0 / GPLv3 / MIT / Unlicense |
+| **0b.** | Documentation | We will provide both **inline documentation** of the code and a basic **tutorial** that explains how a user can (for example) spin up one of our Substrate nodes and send test transactions, which will show how the new functionality works. |
+| **0c.** | Testing and Testing Guide | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests. |
+| **0d.** | Docker | We will provide a Dockerfile(s) that can be used to test all the functionality delivered with this milestone. |
+| 0e. | Article | We will publish an **article**/workshop to explain.
```

Comment:
Given that you have an "education" deliverable in each milestone, I'd say it makes sense to remove this article. Focus on the education part instead and, ideally, publish an article introducing the project separately for M2.

Reply:
> Given that you have an "education" deliverable in each milestone, I'd say it makes sense to remove this article. Focus on the education part instead and, ideally, publish an article introducing the project separately for M2.

thanks, i have modified it according to your suggestion